spec/datarockets/style/cop/layout/array_alignment_extended_spec.rb

```diff
@@ -11,17 +9,29 @@
                           "Width" => indentation_width
                         })
   end
+
   let(:indentation_width) { 2 }
 
+  let(:aligned_array_with_first_element_on_new_line) do
```

Comment:
@nikitasakov just one tip: you can isolate let in the context instead of using so complex names

Reply:
@nikitasakov would you like to make PP tomorrow and make some re-structurization for that test file (for now I don't understand this tests and don't understand where we're using each let veriable)

Reply:
@roman-dubrovsky Yeah, it would be cool. But today a bit inconvenient for me to do PP. Can we schedule it for tomorrow? What time is suites for you?

Reply:
@nikitasako what about 3pm or 4pm tomorrow?

Reply:
or after 9pm?

Reply:
Scheduled at 3pm ✅